packages/core/src/Modules/ComplaintsPolicy/Components/complaints-policy-content.jsx

```diff
@@ -123,178 +103,78 @@ const getGeneralQueriesText = landing_company_shortcode => {
-                            <a
-                                key={2}
-                                className='link link--orange'
-                                rel='noopener noreferrer'
-                                target='_blank'
-                                href='https://www.financialarbiter.org.mt/'
-                            />,
-                        ]}
-                    />
-                );
-
-                if (is_uk) {
-                    texts.push(
-                        <Localize
-                            key={texts.length}
-                            i18n_default_text='<0/><1/>If you reside in the UK and you are unhappy with our response you may escalate your complaint to the <2>Financial Ombudsman Service</2>.'
-                            components={[
-                                <br key={0} />,
-                                <br key={1} />,
-                                <a
-                                    key={2}
-                                    className='link link--orange'
-                                    rel='noopener noreferrer'
-                                    target='_blank'
-                                    href='https://www.financial-ombudsman.org.uk/'
-                                />,
-                            ]}
-                        />
-                    );
-                }
-            }
-            break;
-        }
-    }
-
-    return texts;
-};
-
-const getSubmissionOfAComplaintText = landing_company_shortcode => (
-    <React.Fragment>
+    texts.push(
         <Localize
-            i18n_default_text='To file a complaint about our service, send an email to <0>complaints@deriv.com</0> and state your complaint in detail. Please submit any relevant screenshots of your trading or system for our better understanding.'
+            key={0}
+            i18n_default_text='If you are not satisfied with the outcome, you can escalate your complaint to the <0>Financial Commission</0>.'
             components={[
                 <a
                     key={0}
                     className='link link--orange'
                     rel='noopener noreferrer'
                     target='_blank'
-                    href='mailto:complaints@deriv.com'
+                    href='https://financialcommission.org/resolving-a-dispute/how-to-file-a-complaintdispute/'
                 />,
             ]}
         />
-        {landing_company_shortcode === 'malta' && (
+    );
+
+    if (landing_company_shortcode === 'maltainvest') {
+        texts.push(
             <Localize
-                i18n_default_text=' You may also call <0>+447723580049</0> to place your complaint.'
+                key={texts.length}
+                i18n_default_text='<0/><1/>You may also raise your unresolved dispute to the <2>Office of the Arbiter for Financial Services</2>.'
                 components={[
+                    <br key={0} />,
```

Comment:
[[The br tag is a linebreak tag and used it be very common. However this is no longer the correct way to do it (but you'll still see it around.) It will occasionally be useful and you'll see it in documentation sometimes so I wanted you to know that it exists, it works, and it's not the preferred way of doing linebreaks.](url)](https://btholt.github.io/complete-intro-to-web-dev-v3/lessons/html/types-of-tags#:~:text=The%20br%20tag,of%20doing%20linebreaks.)

Reply:
Thank you, @niloofar-deriv. I've removed some code but the one you shared isn't part of my changes and isn't related to the scope of this card. Let's address it in a separate PR.

Reply:
Alrighty I just put it there later we can replace that